R/plot2.R

```diff
@@ -432,6 +440,30 @@ plot2.default = function(
         }
       )
     )
+  } else if (type %in% "pr") {
+    invisible(
+      lapply(
+        seq_along(split_data),
+        function(i) {
+          segments(
+            x0 = seq_along(split_data[[i]]$x),
+            y0 = split_data[[i]]$ymin,
+            x1 = seq_along(split_data[[i]]$x),
+            y1 = split_data[[i]]$ymax
+          )
+          points(
+            x = split_data[[i]]$x,
+            y = split_data[[i]]$y,
+            col = col[i],
+            type = "p",,
+            pch = pch[i],
+            lty = lty[i]
+          )
```

Comment:
I might be missing something, but what's the rational for repeating the points component here instead of of just doing  `if type %in% c("p", "pointrange")` at line 411 above?

Reply:
https://github.com/grantmcdermott/plot2/pull/35/commits/a90b17362c8b122e8b8d437b370ede719f72f509